test/e2e/data/infrastructure-vsphere-govmomi/main/clusterclass/patch-vsphere-template.yaml

```diff
@@ -14,7 +14,7 @@
             {{- else -}}{{- if eq .builtin.controlPlane.version "v1.29.0" -}}
```

Comment:
Also update the doc above. Maybe in a way that we don't have to keep the version up-to-date (same for the other file)